fcrepo-persistence-common/src/main/java/org/fcrepo/persistence/common/ResourceHeadersImpl.java

```diff
@@ -0,0 +1,264 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.persistence.common;
+
+import java.net.URI;
+import java.time.Instant;
+import java.util.Collection;
+
+import org.fcrepo.kernel.api.models.ResourceHeaders;
+
+/**
+ * Common implementation of resource headers
+ *
+ * @author bbpennel
+ */
+public class ResourceHeadersImpl implements ResourceHeaders {
+
+    private String id;
+
+    private String parent;
+
+    private String stateToken;
+
+    private String interactionModel;
+
+    private String mimeType;
+
+    private String filename;
+
+    private Long contentSize;
+
+    private Collection<URI> digests;
+
+    private String externalUrl;
+
+    private String externalHandling;
+
+    private Instant createdDate;
+
+    private String createdBy;
+
+    private Instant lastModifiedDate;
+
+    private String lastModifiedBy;
+
+    /**
+     * @return the id
+     */
+    @Override
+    public String getId() {
+        return id;
+    }
+
+    /**
+     * @param id the id to set
+     */
+    public void setId(final String id) {
+        this.id = id;
+    }
+
+    /**
+     * @return the parent
+     */
+    @Override
+    public String getParent() {
+        return parent;
+    }
+
+    /**
+     * @param parent the parent to set
```

Comment:
Add description

Reply:
Are these triggering warnings? These are generated javadoc comments, i'm not sure if we need to add descriptions for simple getters/setters beyond the return/params

https://stackoverflow.com/questions/1028967/simple-getter-setter-comments